ThinBridge/ThinBridge
# Install policy templates

# Which issue(s) this PR fixes:

N/A

# What this PR does / why we need it:

This prepares policy template of Chrome and Edge to test Manifest V3 addons more easier.

# How to verify the fixed issue:

Setup the testing environment and confirm that policy templates are installed.

## The steps to verify:

1. `cd doc/verify/windows-10-21H2` or some folder containing Terraform configs.
2. `make`
3. Log in to the prepared environment.
4. Start gpedit.

## Expected result:

There are "Google" and "Microsoft Edge" items under the "Administrative Templates" section.
